docs/examples/popovereducational/customContent.js

```diff
@@ -0,0 +1,44 @@
+// @flow strict
+import { type Node, useRef, useEffect, useState } from 'react';
+import { Box, Flex, IconButton, Button, Text, PopoverEducational } from 'gestalt';
+
+export default function Example(): Node {
+  const [openA, setOpenA] = useState(false);
```

Comment:
nit: the `A` suffix isn't necessary when there's nothing else to differentiate it from